### PR TITLE
Fixed issue #16431: Overview of un-installed themes shows incorrectly parsed theme description - LS4

### PR DESCRIPTION
**Pull request by gabrieljenik**
For uninstalled themes, the description was taken directly from the config metadata directly. No parsing done.

Added templateManifest::getDescription() method for parsing descriptions on XML file. 
This is now used on the view.

### CODE REVIEW

**Comment by sushmanadendlalimesurvey** [comment]
Tested the issue by pulling the PR. The description of uninstalled themes is properly displayed.

File: application/models/TemplateManifest.php [hunk 1]
```diff
@@ -1434,6 +1434,33 @@ public static function getFontDropdownOptions(){
         return $fontOptions;
     }
 
+    /**
+     * Twig statements can be used in Theme description
+     * Override method from TemplateConfiguration to use the description from the XML
```
Comment (olleharstedt) [inline application/models/TemplateManifest.php:1439]: Missing `@return string`.

File: application/models/TemplateManifest.php [hunk 2]
```diff
@@ -1434,6 +1434,33 @@ public static function getFontDropdownOptions(){
...
+        try {
+            $sDescription = App()->twigRenderer->convertTwigToHtml($this->config->metadata->description);
+        } catch (\Exception $e) {
+          // It should never happen, but let's avoid to anoy final user in production mode :)
+            if (YII_DEBUG) {
+                App()->setFlashMessage(
+                    "Twig error in template " .
+                    $this->sTemplateName .
+                    " description <br> Please fix it and reset the theme <br>" .
+                    $e,
```
Comment (olleharstedt) [inline application/models/TemplateManifest.php:1456]: `$e->getMessage()`?

File: application/models/TemplateManifest.php [hunk 3]
```diff
@@ -1434,6 +1434,33 @@ public static function getFontDropdownOptions(){
...
+                    'error'
+                );
+            }
+        }
+
+          return $sDescription;
```
Comment (olleharstedt) [inline application/models/TemplateManifest.php:1462]: Indentation seems off?

File: application/models/TemplateManifest.php (hunk 2 above)
Comment (gabrieljenik) [reply application/models/TemplateManifest.php]: Fixed

File: application/models/TemplateManifest.php (hunk 3 above)
Comment (gabrieljenik) [reply application/models/TemplateManifest.php]: Fixed

File: application/models/TemplateManifest.php (hunk 1 above)
Comment (gabrieljenik) [reply application/models/TemplateManifest.php:1439]: Fixed

**Pull request merged**